pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -64,6 +64,7 @@ let
     else if targetPlatform.system == "aarch64-linux"  then "${libc_lib}/lib/ld-linux-aarch64.so.1"
     else if targetPlatform.system == "powerpc-linux"  then "${libc_lib}/lib/ld.so.1"
     else if targetPlatform.isMips                     then "${libc_lib}/lib/ld.so.1"
+    else if targetPlatform.system == "riscv64-linux"  then "${libc_lib}/lib/ld-linux-riscv64-lp64d.so.1"
```

Comment:
Maybe we want to add targetPlatform.isRiscv64 ?

Reply:
@SuperSandro2000 
> Maybe we want to add targetPlatform.isRiscv64 ?

`lp64d` (part of `so` name) is just one of riscv64 ABIs. User-defined riscv64 platform may use other ABIs.

Reply:
Any user defined one would still use `system = "riscv64-linux"` however. Extra ABI info should go inside `targetPlatform.parsed.abi` somehow. See https://github.com/NixOS/nixpkgs/blob/master/lib/systems/parse.nix#L307-L361

Reply:
We only have `isRiscV` instead of  `isRiscV64`. Maybe we could use `"${libc_lib}/lib/ld-linux-riscv*.so.1"` like what we did for ARM?

Reply:
Yeah that should be possible I think.